src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,20 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import React from 'react';
+import { User } from '../../types/User';
+
+interface UserInfoProps {
+  user: User;
+}
+
+export const UserInfo: React.FC<UserInfoProps> = ({ user }) => {
+  return (
+    <a className="UserInfo" href={`${user.email}`}>
+      <p>
+        mailto:
+        {user.email}
+      </p>
+      <p>
+        {user.name}
+      </p>
```

Comment:
And you don't need to write the email explicitly. Just make the correct link with the name as a title
```suggestion
      {name}
```